src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/CommonTestTimeouts.cs

```diff
@@ -42,5 +42,35 @@ public static class CommonTestTimeouts
         /// Dumps (especially full dumps) can be quite large and take a significant amount of time to transfer.
         /// </remarks>
         public static readonly TimeSpan DumpTimeout = TimeSpan.FromMinutes(3);
+
+        /// <summary>
+        /// Default timeout for logs collection.
+        /// </summary>
+        public static readonly TimeSpan LogsTimeout = TimeSpan.FromMinutes(3);
+
+        /// <summary>
+        /// Default logs collection duration.
+        /// </summary>
+        public static readonly TimeSpan LogsDuration = TimeSpan.FromSeconds(10);
+
+        /// <summary>
+        /// Default timeout for HTTP API calls
+        /// </summary>
+        public static readonly TimeSpan HttpApi = TimeSpan.FromSeconds(15);
+
+        /// <summary>
+        /// Timeout for waiting for a collection rule to complete.
+        /// </summary>
+        public static readonly TimeSpan CollectionRuleCompletionTimeout = TimeSpan.FromSeconds(30);
```

Comment:
Some of these timeouts can probably be moved back to TestTimeouts from CommonTestTimeouts; had moved them when experimenting but didn't put them back.